### DISCUSSION TITLE
Bearer token support

### DISCUSSION THREAD

**Discussion by gautamsi** [keystonejs/keystone#6275] [Feature requests]
I had to struggle with and figured finally that there is no `Bearer` token support in v6. It would be good to have this support to enable hybrid mobile clients (ionic/React Native/Cordova etc.) work without any native plugins. 

I had problem with ios devices which prevents cross site cookies, android was also struggle in correct session information.

I was however able to clone the session logic and check for header by doing this I am sure it will be helpful for others, also adding a PR soon

```diff
async get({ req }) {
-  if (!req.headers.cookie) return;
-  let cookies = cookie.parse(req.headers.cookie);
-  if (!cookies[TOKEN_NAME]) return;
-  try {
-      return await Iron.unseal(cookies[TOKEN_NAME], secret, ironOptions);
+  cont cookies = cookie.parse(req.headers.cookie || '');
+  const bearer = req.headers.authorization?.replace('Bearer ', '');
+  const token = bearer || cookies[TOKEN_NAME];
+  if (!token) return;
+  try {
+    return await Iron.unseal(token, secret, ironOptions);
  } catch (err) {}
},
```

**Comment by bladey** [comment]
Closed with https://github.com/keystonejs/keystone/pull/6276
